src/core/sent_packet_metadata.h

```diff
@@ -103,10 +119,17 @@ typedef struct QUIC_SENT_PACKET_METADATA {
 
     uint64_t PacketId;
     uint64_t PacketNumber;
+
     uint32_t SentTime; // In microseconds
+
     uint16_t PacketLength;
+
     uint8_t PathId;
 
+    uint64_t TotalBytesSentThen;
+
+    LAST_ACKED_PACKET_INFO LastAckedPacketInfo;
```

Comment:
This is a significant increase in the amount of memory required to track every single sent packet. Like 32 to 40 extra bytes multiplied by every outstanding packet. Is this absolutely necessary? It seems to really be a shame that we'd have to include this when cubic doesn't even use it.

BTW, at the very least, `TotalBytesSentThen` (which I don't even understand based on the variable name) needs to be put after `PacketNumber` for the best padding/packing. `LastAckedPacketInfo` should also be at the top if we keep it.

Reply:
fixed.